script.js

```diff
@@ -0,0 +1,21 @@
+function Newbook() {
+  const Title = document.getElementById('Title');
+  const Author = document.getElementById('Author');
+  const newdiv = document.createElement('div');
+  newdiv.setAttribute('id', 'book_infos');
+  const par1 = document.createElement('p');
+  const par2 = document.createElement('p');
+  const buutt = document.createElement('button');
+  buutt.textContent = 'remove';
+  buutt.setAttribute('id', 'rem');
+  par1.appendChild(document.createTextNode(Title.value));
```

Comment:
- [ ] Add tiltes for the list, one for "Title" of the book and other for "Author" of the book. You can achieve this in JS but will be easier in HTML because it's a constant element in your app.